content/en/xtc/xtc-release-notes/v80.md

```diff
@@ -0,0 +1,39 @@
+---
+title: XTC v80
+linkTitle: XTC v80
+
+weight: 136
+type: docs
+
+date: 2024-03-18
+
+description: 2024-03-19
+
+sitemap:
+  changefreq: never
+  priority: 0.1
+---
+
+## Platform
+
+* API v2 (still in preview)
+    * The load test "utility" endpoint `/public/api/v2/{org}/projects/{project}/load-tests/{id})` has been extended with a variant to update the name and the description of a load test. See the API Explorer for details.
+    * The endpoints for querying the list of projects and the list of load tests now support paging, filtering, and sorting via query string parameters. Again, see the API Explorer for details.
+* Date/time input fields are now accompanied by buttons to quickly set a time value relative to the current time (now, +30min, +1h, +2h, +3h, +6h).
+* The minimum width of all tables has been increased so that tables no longer shrink to fit the screen, but instead provide a horizontal scroll bar. Especially on small screens, this may require the user to scroll through tables, but the information displayed is often displayed in full length, rather than truncated to a few characters.
```

Comment:
```suggestion
* The minimum width of all tables has been increased so that tables no longer shrink to fit the screen, but provide a horizontal scroll bar instead. Especially on small screens, this may require the user to scroll through tables. But the shown information is often displayed in full length rather than truncated to a few characters.
```